index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw=="
+    crossorigin="anonymous" />
+    <link rel="stylesheet" href="/style.css">
+    <title>Midori | Home</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header-items">
+            <div class="nav-brand">Midori</div>
+              <div class="input-icon icons-left">
+                <span class="material-icons">search</span> 
+                <input class="input" type="" id="" name="" placeholder="search"/> 
```

Comment:
name is empty here, if not necessary better to remove it and rest all looks good